.eslintrc.js

```diff
@@ -1,22 +1,32 @@
 module.exports = {
   root: true,
-  parser: 'babel-eslint',
+  parser: '@typescript-eslint/parser',
   parserOptions: {
     ecmaVersion: 2018,
-    sourceType: 'module'
+    sourceType: 'module',
+    project: './tsconfig.json',
+    // allows eslint from any dir
+    tsconfigRootDir: __dirname,
+    sourceType: 'module',
```

Comment:
Looks like this key wound up in there twice

Reply:
Whoops, sorry probably did my suggestion wrong 😢

Reply:
Fixed!